src/front/app/features/create/components/validate.js

```diff
@@ -0,0 +1,19 @@
+const validate = (values) => {
+    const errors = {};
+
+    if (!values.title) {
+        errors.title = 'Title is required';
+    }
+
+    if (!values.description) {
+        errors.description = 'Description is required';
+    }
+
+    if (values.category === '-1' || !values.category) {
```

Comment:
Move checking for empty category as firs argument. It is better because in that case "values.category === '-1'" will be executed only if category is not empty.